From 3bbbbf3da469f457881b5af902eb0b89b95189a2 Mon Sep 17 00:00:00 2001
From: Samuel Georges <sam@daftspunk.com>
Date: Tue, 10 Oct 2017 10:12:33 +1100
Subject: [PATCH] Remove SVG from image types

SVG files should not be treated as images - especially when coming to uploads. An SVG file can contain arbitrary HTML data as well as event handlers in native elements
Refs: https://html5sec.org/#svg
Original report by: Ishaq Mohammed
---
 src/Filesystem/Definitions.php | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/Filesystem/Definitions.php b/src/Filesystem/Definitions.php
index baa4d9812..63660e464 100644
--- a/src/Filesystem/Definitions.php
+++ b/src/Filesystem/Definitions.php
@@ -182,8 +182,7 @@ protected function imageExtensions()
             'bmp',
             'png',
             'webp',
-            'gif',
-            'svg'
+            'gif'
         ];
     }
 
